src/mixer.c

```diff
@@ -59,33 +54,52 @@ struct loop_data
 	uint32 epilogue[LOOP_EPILOGUE];
 };
 
+/* Mixers array index:
+ *
+ * bit 0: 0=8 bit sample, 1=16 bit sample
+ * bit 1: 0=mono sample, 1=stereo sample
+ * bit 2: 0=mono output, 1=stereo output
+ * bit 3: 0=unfiltered, 1=filtered
+ */
+
+#define FLAG_16_BITS	0x01
+#define FLAG_STEREO	0x02
+#define FLAG_STEREOOUT	0x04
+#define FLAG_FILTER	0x08
+#define FLAG_ACTIVE	0x10
+/* #define FLAG_SYNTH	0x20 */
```

Comment:
Maybe it's time to retire this flag, as there is absolutely no indication what it's even supposed to do.

Reply:
I have no clue either—I *assume* it's meant to signal to the mixer that some notes shouldn't be played as samples, but instead should be passed to a synthesizer. It seems like a mistake to me to send notes/effects to the synthesizer *after* the `read_event` functions and player.c have processed them but whatever.

I was just going to leave figuring this flag out until I add a new OPL player, whenever that is.

Reply:
Looking back at a really old code revision, this does indeed seem to be the only remnant from OPL emulation. Maybe it has to be this way in libxmp but in OpenMPT such voices wouldn't even make their way into the mixer, so this seems strange to me.